sympy/functions/special/error_functions.py

```diff
@@ -1433,6 +1433,12 @@ def _eval_aseries(self, n, args0, x, logx):
 
         return super(expint, self)._eval_aseries(n, args0, x, logx)
 
+    def _eval_rewrite_as_Integral(self, *args, **kwargs):
+        from sympy.integrals.integrals import Integral
+        n, x = self.args
+        t = Dummy('t')
+        return Integral((t**-n) * (exp(-t*x)), (t, 1, S.Infinity))
```

Comment:
```suggestion
        return Integral(t**-n * exp(-t*x), (t, 1, S.Infinity))
```

I think that you can remove some unnecessary parenthesis